.idea/codestyles/Project.xml

```diff
@@ -29,7 +29,16 @@
     <JetCodeStyleSettings>
       <option name="PACKAGES_TO_USE_STAR_IMPORTS">
         <value>
-          <package name="kotlinx.android.synthetic" withSubpackages="true" static="false" />
+          <package name="kotlinx.android.synthetic" alias="false" withSubpackages="true" />
```

Comment:
Do you know why this file is changing now? We should revert.

Reply:
I have no idea what is causing this.  I opened up some preferences this morning, but I didn't change anything.  I will have to keep an eye out for it.